src/routing.js

```diff
@@ -107,26 +104,27 @@ export function getAppPath({
     return `/${action}`
   }
 
-  if (!fullDao) {
-    return `/${search}`
+  // Only keep the DAO name if it ends in aragonid.eth
```

Comment:
It seems a bit odd that we're doing this here in routing, but moving the redirect out into the app itself.

Reply:
The redirect used to be done here, but I don’t think this function should have any side effect. But I don’t think it should stay in `App` either, I was thinking of moving all of the `history` / `window.location` handling in the routing provider that remain to be added.

Reply:
> I was thinking of moving all of the history / window.location handling in the routing provider that remain to be added

Yes! This sounds like exactly what we need!